apps/marketplace/actions/supplierActions.js

```diff
@@ -0,0 +1,16 @@
+import dmapi from '../services/apiClient'
+
+const findSuppliers = keyword => {
+  const params = {
+    keyword
+  }
+  return dmapi({ url: `/suppliers/search`, params }).then(response => {
+    if (!response || response.error) {
+      throw response.errorMessage
+    } else {
+      return response.data
+    }
+  })
+}
+
+export default findSuppliers
```

Comment:
Needed to make this default as eslint prefers a file with only one export to use `default` apparently.